buildSrc/src/main/groovy/Dependencies.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2019 Connexta, LLC
+
+Released under the GNU Lesser General Public License version 3; see
+https://www.gnu.org/licenses/lgpl-3.0.html
+*/
+// Default Package
+class Dependencies {
```

Comment:
❓Why not just stick the version right in the dependency definition for these that don't share a version with any other dependency?

Reply:
Mostly for consistency reasons. It'd be weird to be able to override a version on some dependencies but not all of them.

for example if this works
`Dependencies.foo("1.2.3")`
but this doesn't
`Dependencies.bar("1.2.3")`
I feel that'd be werid

Reply:
I don't understand your response. I was suggesting doing this:
```groovy
static def commonsIO = "commons-io:commons-io:2.6"
```
```gradle
implementation(Dependencies.commonsIO())
```

Reply:
Well there's two things here to describe, the first is the use of closures and the second is the having the version in the same spot.

the reason why I had it setup with  
```
static def commonsIO = { it=Versions.commonsIO -> "commons-io:commons-io:${it}" }
```
was to allow someone to override a version for a hopefully good reason.
`Dependencies.commonIO("1.2.3")`
would use 1.2.3 instead of the actual version.

For the second part, personal preference and consistency so all the versions are together. I think it'd look strange to have some versions be variables and some be strings

Reply:
Not a blocker for this PR. You wouldn't ever use `Versions.commonsIO` outside of `Dependencies.gradle`, right? Moving the version to `Dependencies.gradle` would enforce that.

Reply:
I did some fiddling with the groovy files and it doesn't look like there's a way to make something truly private in groovy. If it's private you can still use it in another file, but intellij will show a warning saying you're exceeding the access rights

Reply:
Why would you have to make it private. Would this work?
```groovy
static def commonsIO = { it="2.6" -> "commons-io:commons-io:${it}" }
```
This would maintain the ability to overwrite the version if you wanted to, right?
```gradle
implementation(Dependencies.commonIO("1.2.3"))
```

Reply:
I was going trying to make the variables private to keep them around. Doing that does work but I don't like how it looks since we'd be mixing strings and variables. That's my take on it
```
  static def commonsIO = { it="2.6" -> "commons-io:commons-io:${it}" }
  static def springBootWebStarter = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-web:${it}" }
```